README.md

```
@@ -51,19 +71,21 @@ git-stack is a developer productivity tool that makes stacked pull requests simp
-  1. Fetching from remote...
-  2. Syncing 2 stack(s):
-
-  📦 Syncing stack: feature-auth
-    ✓ Stack rebased successfully
-  ✅ All stacks synchronized successfully!
-  ```
+   ```bash
+   # From main - sync all stacks
+   git-stack sync
+   ```
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)